docs/man/rpm.8.md

```diff
@@ -1,1077 +1,841 @@
 
-**rpm** {**\--reinstall**} \[**install-options**\] *PACKAGE\_FILE \...*
+*rpm* {*-e|--erase*} [*erase-options*] *PACKAGE_NAME ...*
 
-**rpm** {**-e\|\--erase**} \[**\--allmatches**\] \[**\--justdb**\]
-\[**\--nodb**\] \[**\--nodeps**\] \[**\--noscripts**\] \[**\--notriggers**\]
-\[**\--test**\] *PACKAGE\_NAME \...*
+*rpm* *--restore* [*select-options*] *PACKAGE_NAME ...*
 
-MISCELLANEOUS
--------------
+## MISC OPERATIONS
 
-**rpm** **\--showrc**
+*rpm* *--querytags*
 
-**rpm** **\--restore** \[**select-options**\]
+*rpm* *--showrc*
 
-select-options
---------------
+# DESCRIPTION
 
-\[*PACKAGE\_NAME*\] \[**-a,\--all \[***SELECTOR*\]\] \[**-f,\--file**
-*FILE*\] \[**\--path** *PATH*\] \[**-g,\--group** *GROUP*\] \[**-p,\--package**
-*PACKAGE\_FILE*\]
-\[**\--tid** *TID*\] \[**\--querybynumber** *HDRNUM*\]
-\[**\--triggeredby** *PACKAGE\_NAME*\] \[**\--whatprovides**
-*CAPABILITY*\] \[**\--whatrequires** *CAPABILITY*\]
-\[**\--whatrecommends** *CAPABILITY*\] \[**\--whatsuggests**
-*CAPABILITY*\] \[**\--whatsupplements** *CAPABILITY*\]
-\[**\--whatenhances** *CAPABILITY*\] \[**\--whatobsoletes**
-*CAPABILITY*\] \[**\--whatconflicts** *CAPABILITY*\]
-
-query-options
--------------
-
-General: \[**\--changelog**\] \[**\--changes**\] \[**\--dupes**\]
-\[**-i,\--info**\] \[**\--last**\] \[**\--qf,\--queryformat**
-*QUERYFMT*\] \[**\--xml**\] \[**\--json**\]
-
-Dependencies: \[**\--conflicts**\] \[**\--enhances**\]
-\[**\--obsoletes**\] \[**\--provides**\] \[**\--recommends**\]
-\[**-R,\--requires**\] \[**\--suggests**\] \[**\--supplements**\]
-
-Files: \[**-c,\--configfiles**\] \[**-d,\--docfiles**\] \[**\--dump**\]
-\[**\--fileclass**\] \[**\--filecolor**\]
-\[**\--fileprovide**\]\[**\--filerequire**\] \[**\--filecaps**\]
-\[**\--filesbypkg**\] \[**-l,\--list**\] \[**-s,\--state**\]
-\[**\--noartifact**\] \[**\--noghost**\] \[**\--noconfig**\]
-
-Scripts and triggers: \[**\--filetriggers**\] \[**\--scripts**\]
-\[**\--triggers,\--triggerscripts**\]
-
-verify-options
---------------
-
-\[**\--nodeps**\] \[**\--nofiles**\] \[**\--noscripts**\]
-\[**\--nodigest**\] \[**\--nosignature**\] \[**\--nolinkto**\]
-\[**\--nofiledigest**\] \[**\--nosize**\] \[**\--nouser**\]
-\[**\--nogroup**\] \[**\--nomtime**\] \[**\--nomode**\]
-\[**\--nordev**\] \[**\--nocaps**\]
-
-install-options
----------------
-
-\[**\--allfiles**\] \[**\--badreloc**\] \[**\--excludepath** *OLDPATH*\]
-\[**\--excludedocs**\] \[**\--force**\] \[**-h,\--hash**\]
-\[**\--ignoresize**\] \[**\--ignorearch**\] \[**\--ignoreos**\]
-\[**\--includedocs**\] \[**\--justdb**\] \[**--nodb**\] \[**\--nodeps**\]
-\[**\--nodigest**\] \[**\--noplugins**\] \[**\--nocaps**\]
-\[**\--noorder**\] \[**\--noverify**\] \[**\--nosignature**\]
-\[**\--noscripts**\] \[**\--notriggers**\] \[**\--oldpackage**\]
-\[**\--percent**\] \[**\--prefix** *NEWPATH*\] \[**\--relocate**
-*OLDPATH***=***NEWPATH*\] \[**\--replacefiles**\]
-\[**\--replacepkgs**\] \[**\--test**\]
```

Comment:
I guess this deserves a special note (perhaps in the commit message even): deleting these kind of option summaries was very much intentional, maintaining such things is a lot of error-prone work - spotted several errors/inconsistencies while doing this conversion. The effort is better spent ensuring all options are documented once, and that documentation is correct.